src/state/migrations/index.ts

```diff
@@ -1,125 +1,36 @@
-  36: clearTxHistory,
-  37: clearPortfolio,
-  38: clearOpportunities,
-  39: clearAssets,
-  40: clearAssets,
-  41: clearAssets,
-  42: clearAssets,
-  43: clearAssets,
-  44: clearAssets,
-  45: clearAssets,
-  46: clearAssets,
-  47: clearAssets,
-  48: clearAssets,
-  49: clearAssets,
-  50: clearPortfolio,
-  51: clearAssets,
-  52: clearPortfolio,
-  53: clearAssets,
-  54: clearAssets,
-  55: clearAssets,
-  56: clearPortfolio,
-  57: clearAssets,
-  58: clearAssets,
-  59: clearAssets,
-  60: clearAssets,
-  61: clearOpportunities,
-  62: clearAssets,
-  63: clearAssets,
-  64: clearPortfolio,
-  65: clearAssets,
-  66: clearAssets,
-  67: clearAssets,
-  68: clearAssets,
-  69: clearAssets,
-  70: clearAssets,
-  71: clearAssets,
-  72: clearAssets,
-  73: clearAssets,
-  74: clearAssets,
-  75: clearAssets,
-  76: clearAssets,
-  77: clearAssets,
-  78: clearAssets,
-  79: clearAssets,
-  80: clearAssets,
-  81: clearAssets,
-  82: clearAssets,
-  83: clearAssets,
-  84: clearAssets,
-  85: clearAssets,
-  86: clearAssets,
-  87: clearWalletConnectWalletsMetadata,
-  88: clearAssets,
-  89: clearAssets,
-  90: clearAssets,
-  91: clearAssets,
-  92: clearTxHistory,
-  93: clearAssets,
-  94: clearAssets,
-  95: clearTxHistory,
-  96: clearTxHistory,
-  97: clearAssets,
-  98: clearAssets,
-  99: clearAssets,
-  100: clearAssets,
-  101: clearAssets,
-  102: clearAssets,
-  103: clearAssets,
-  104: clearTxHistory,
-  105: clearPortfolio,
-  106: clearOpportunities,
-  107: clearAssets,
-  108: clearAssets,
-  109: clearAssets,
-  110: clearAssets,
-  111: clearAssets,
-  112: clearAssets,
-  113: clearAssets,
-}
+export const clearTxHistoryMigrations = {
+  // Uncomment me when introducing the first migration for this slice
+  // 0: clearTxHistory,
+} as unknown as Omit<MigrationManifest, '_persist'>
+
+export const clearOpportunitiesMigrations = {
+  // Uncomment me when introducing the first migration for this slice
+  // 0: clearOpportunities,
+} as unknown as Omit<MigrationManifest, '_persist'>
+
+export const clearPortfolioMigrations = {
+  // Uncomment me when introducing the first migration for this slice
+  // 0: clearPortfolio,
+} as unknown as Omit<MigrationManifest, '_persist'>
+
+export const clearNftsMigrations = {
+  // Uncomment me when introducing the first migration for this slice
+  // 0: clearNfts,
+} as unknown as Omit<MigrationManifest, '_persist'>
+
+export const clearAssetsMigrations = {
+  // Uncomment me when introducing the first migration for this slice
+  // 0: clearAssets,
+} as unknown as Omit<MigrationManifest, '_persist'>
+
+export const clearMarketDataMigrations = {
+  // Uncomment me when introducing the first migration for this slice
+  // 0: clearMarketData,
+} as unknown as Omit<MigrationManifest, '_persist'>
+
+export const clearSnapshotMigrations = {
+  // Uncomment me when introducing the first migration for this slice
+  // 0: clearSnapshot,
+} as unknown as Omit<MigrationManifest, '_persist'>
```

Comment:
[blocking question]: What is the default version and should we have it here so the first migration doesnt fail?

Reply:
We should not - that will be automagically injected by redux-persist (tested with 0th migrations)

Reply:
i.e no migration:

<img width="519" alt="image" src="https://github.com/user-attachments/assets/ad4f0713-97b0-46f7-a2ef-2391906a5f3c">

0th migration:

<img width="531" alt="image" src="https://github.com/user-attachments/assets/cd1e9ddc-83b9-41b1-ab01-0fb98a8412ef">